src/ui/render.ts

```diff
@@ -56,6 +76,37 @@ export default class UI {
     tableContainerDom.appendChild(this.tableBodyDom);
   }
 
+  createToolBar() {
+    if (this.toolbarOptions) {
+      //Element
+      if (this.toolbarOptions.element != null) {
+        this.toolbarOptions.element.appendChild(this.toolBarDom);
+      } else {
+        this.tableEl.insertBefore(this.toolBarDom, this.tableEl.firstChild);
+      }
+
+      const toolbarContent = this.toolbarOptions.items!;
+      for (let i = 0; i < toolbarContent.length; i++) {
+        const toolbarItem = document.createElement("i");
+        toolbarItem.classList.add("lightSheet_toolbar_item");
+        toolbarItem.classList.add("material-symbols-outlined");
+        toolbarItem.textContent = toolbarContent[i];
+        this.toolBarDom.appendChild(toolbarItem);
+      }
+
+      //showToolBar
+      this.showToolBar(this.toolbarOptions.showToolbar!);
+    }
+  }
+
+  showToolBar(isShown: boolean) {
+    if (isShown) {
+      this.toolBarDom.style.display = "flex";
+    } else {
+      this.toolBarDom.style.display = "none";
+    }
+  }
```

Comment:
If we skip creating `this.toolBarDom` as I suggest in an earlier comment, this method should check for `toolBarOptions`. 
The method can also be simplified to `this.toolBarDom.style.display = isShown ? "flex" : "none";`